packages/core/integration-tests/test/babel.js

```diff
@@ -313,6 +324,50 @@ describe('babel', function() {
     await outputFS.rimraf(path.join(fixtureDir, 'dist'));
   });
 
+  it('should support building with default babel config when running parcel globally', async function() {
+    let tmpDir = tempy.directory();
+    let distDir = path.join(tmpDir, 'dist');
+    await fs.ncp(
```

Comment:
should we maybe copy this into the memory fs instead to avoid writing to disk during the tests?